### ISSUE TITLE
All @facebook identity proofs showing as "unreachable", including my own

### ISSUE THREAD

**Issue by reedriley** [keybase/keybase-issues#3556]
When I click on the proofs; I'm taken to a correct looking identity proof on the user's Facebook  profile.  But within the Keybase app; all @facebook proofs are showing as "UNREACHABLE.

Reproduces on at least:
* The iOS client (version 4.4.2)
* The OS X client (version 4.4.2-20190912194049+4931602c61)
* The CLI (version 4.4.2-20190912194049+4931602c61)

The CLI shows this error message:
> "<user>" on facebook failed: Could not find post text in Facebook's response (code=106)

I'm guessing something about the Facebook response changed, breaking client parsing?

**Comment by mlsteele** [comment]
Yep the Facebook response did change and we made 2 changes in the last few days to try to fix it. What users proofs are show unreachable? Feel free to send me the usernames via keybase chat if you'd prefer.

**Comment by mlsteele** [comment]
I missed your "including my own". Your facebook proof likes like it's ✅ to me.

**Comment by mlsteele** [comment]
Talked out of band. Seems like checking facebook proofs over tor is broken for now due to captchas. A workaround is in our sights.

**Comment by felixbuenemann** [comment]
Thanks for the update, I think I'm running into the same issue.

The log shows that he post ist retrieved with status 200 OK, but the CSS selector doesn't match.

Using Keybase 4.5.0-20190919040131+93e889ab01 on macOS 10.14.6.

If I curl the post url directly it's fine, but through a tor proxy curl also gets a captcha.

So I think @mlsteele is on the right track.

**Comment by pabru** [comment]
Any update/ETA on this?

**Comment by xaviablaza** [comment]
Any updates on this issue? @mlsteele

**Comment by McPringle** [comment]
It's 2022 now and the problem is still the same. The Facebook identity is marked as unreachable but "View proof" works like a charm. The message is public and can be viewed when no user is logged in. Only when accessing using the Tor network, Facebook does not show the message and forwards to a login form instead. The check can't be done using the Tor network. It must be done via "public internet".

**Comment by My1** [comment]
same here. my FB Proof also looks broken on keybase but the URL checks out

**Comment by the-reverend** [comment]
same here. the proof is broken in keybase but the facebook post url is public and viewable even incognito in a browser.

**Comment by mcg1969** [comment]
October 26, 2024 ... still true.
